### PR TITLE
Fix sniffer complaints

### PR DESCRIPTION
**Pull request by nickvergessen**
@rullzer

### CODE REVIEW

File: templates/main.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ]);
 ?>
 
-<div id="app" class="announcementcenter" data-is-admin="<?php if (!empty($_['isAdmin'])) { p(1); } else { p(0); } ?>">
+<div id="app" class="announcementcenter" data-is-admin="<?php p(!empty($_['isAdmin']) ? 1 : 0); ?>">
```
Comment (rullzer) [inline templates/main.php]: Does it find this?
Impressive

File: tests/TestCase.php [hunk 2]
```diff
@@ -24,43 +24,4 @@
...
 abstract class TestCase extends \Test\TestCase {
-	/** @var array */
-	protected $services = [];
-
-	/**
-	 * @param string $name
-	 * @param mixed $newService
-	 * @return bool
-	 */
-	public function overwriteService($name, $newService) {
```
Comment (rullzer) [inline tests/TestCase.php]: Not used?

**Comment by rullzer** [comment]
LGTM

File: tests/TestCase.php (hunk 2 above)
Comment (nickvergessen) [inline tests/TestCase.php]: I added it to `\Test\Testcase` directly months ago, so yeah useless overwriting.

File: templates/main.php (hunk 1 above)
Comment (nickvergessen) [inline templates/main.php]: yeah, but I excluded it in the core PR afterwards. we have to many lines with more then one control-structure.

**Pull request merged**